examples/specs/layer_bar_dual_axis_minmax.vl.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
We already have layer_box_plot_circle as a good example for layer of layer, so I prefer not to include this.